Correct joint limits for s Model

Jntzko opened the pull request:
The upper and lower limits for the joints were identified with the
robotiq_joint_state_publisher for the s model by moving to different poses via the SModelSimpleController.py.
This was necessary in order to use moveit with this urdf out of the box.

Commit:
Correct joint limits for s Model

The upper and lower bounds for the fingers were identified with the
robotiq_joint_state_publisher for the s model and corrected in the
s-model_articulated_macro.xacro for the palm finger joints and in the
s-model_finger_articulated_macro.xacro for the remaining finger joints.

shaun-edwards commented:
Can you clarify what you mean by "using" MoveIt?  Are you planning for the articulated fingers?

Jntzko commented:
I've written an "GripperCommand" action server for the gripper that supports specific open and close positions (https://github.com/TAMS-Group/robotiq_s_model_action_server) and added it to moveit's list of controllers. When using moveit's planning capability to go to these states I get errors from moveit that some joints are out of their joint limits, so I adjusted all joint limits to their respective maximum/minimum. I got the values by moving the hand around with the SModelSimpleController.py.

shaun-edwards commented:
@Jntzko, thanks for the contribution.  +1...merging.
